index.html

```diff
@@ -2825,125 +2827,265 @@ <h3>Working with personal names</h3>
-	<details class="links"><summary>explanations &amp; examples</summary>
+<p>For example, in some cases you may want to identify parts of a name so that you can sort a list of names alphabetically, or address them when contacting them, etc.</p>
+   
+<p>This extra field would also be useful for finding the appropriate name from a long list of name components, and for handling nicknames (which, for example, are commonly used to refer to people in Thailand).</p>
+<p>Sometimes you may opt for separate fields because you want to be able to use part of the name to address the person directly, or refer to them. For example, when Google+ refers to "David's contacts". Or perhaps it's because you want to send them emails with their name at the top. Note that not only may you have problems due to name syntax here, but you also have to account for varying expectations around the world with regards to formality (not everyone is happy for a stranger to call them by their given name). It may be better to ask separately, when setting up a profile for example, how that person would like you to address them.</p>
+
+<div class="req" id="loc_names_relevant_info">
+<p class="advisement">If parts of a person's name are captured separately, ensure that the separate items can capture all relevant information.</p>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#splitting">Strategies for splitting up names</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
 	</div>
 
-	<div class="req" id="loc_names_algorithms">
+<p>For example, don't assume that the order they will provide names in will be given followed by family.</p>
+
+<div class="req" id="loc_names_algorithms">
 	<p class="advisement">Be careful about assumptions built into algorithms that pull out the parts of a name automatically.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#splitting">Strategies for splitting up names</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
-	</div>
+    <details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
+</div>
+
+<p>For example, the v-card and h-card approach of implied “n” optimization could have difficulties with, say, Chinese names. The input form should be as clear as possible when telling people how to specify their name, so that you capture the data you think you need.</p>
```

Comment:
`v-card`? `h-card`? `implied "n" optimization`? Something missing here I suspect.

Reply:
I think it's about the personal names in the [vCard](https://www.rfc-editor.org/info/rfc6350) and [h-card](http://microformats.org/wiki/h-card) formats.

I'm not sure what `implied "n" optimization` means here.

Reply:
Just found this: http://microformats.org/wiki/hcard#Implied_.22N.22_Optimization

Reply:
Yes.